tools/test-harnesses/src/queries/lgId.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * The majority of the code is copied from [dom-testing-library](https://github.com/testing-library/dom-testing-library/blob/bd04cf95a1ed85a2238f7dfc1a77d5d16b4f59dc/src/queries/test-id.ts).
+ * *
+ * [Testing Library](https://testing-library.com/docs/dom-testing-library/api-configuration) provides a way to globally change the testIdAttribute but, unfortunately, it does not suit our needs. We prefer not to modify the testIdAttribute globally as it would impact all tests across the entire repository.
+ *
+ * As an alternative, we're importing the `buildQueries` function directly from [dom-testing-library](https://github.com/testing-library/dom-testing-library/blob/bd04cf95a1ed85a2238f7dfc1a77d5d16b4f59dc/src/queries/test-id.ts) and assigning our custom testing id, `data-lgid`, to it.
+ */
```

Comment:
👍 (though it looks like something odd is happening with the comment formatting... may just be GH though)

Reply:
I had an extra asterisk in there but i think it should be good now